server/bundler/plugin.js

```diff
@@ -30,11 +35,12 @@ ChunkFileNames.prototype.apply = function( compiler ) {
 						"script.onerror = script.onload = script.onreadystatechange = null;",
 						"delete installedChunks[ chunkId ];",
 						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name ) + "[chunkId]||chunkId ]=new Error();",
-						"callback.call( null, " + this.requireFn + ")"
+						"return Promise.resolve();"
```

Comment:
this is in a script.onerror handler, so ... who's picking up the resolved promise? should this be calling `reject` on the promise created and stuck into `installedChunks`?